src/javascript/app/pages/trade/price.js

```diff
@@ -213,11 +233,16 @@ const Price = (() => {
             }
             comment.show();
             error.hide();
-            commonTrading.displayCommentPrice(comment, (currency.value || currency.getAttribute('value')), proposal.ask_price, proposal.payout);
+            if (!/^(LBFLOATCALL|LBFLOATPUT|LBHIGHLOW)$/.test(type)) {
```

Comment:
can we also switch here so it's `if (/^(LBFLOATCALL|LBFLOATPUT|LBHIGHLOW)$/.test(type)) {...} else {...`